### ISSUE TITLE
Prevent clicking "Out of popup"

### ISSUE THREAD

**Issue by ablohme86** [rotorgames/Rg.Plugins.Popup#748]
## 🚀 Feature Requests

Is there a way to prevent the user from clicking outside the popup so it doesnt close?

### Platforms affected (mark all that apply)
- [X ] :iphone: iOS
- [ X] :robot: Android
- [ NA] :checkered_flag: WPF
- [ NA] :earth_americas: UWP
- [ NA] :apple: MacOS
- [ NA] :tv: tvOS
- [ NA] :monkey: Xamarin.Forms

**Comment by FaroukBel** [comment]
Yes, there is a feature called `CloseWhenBackgroundIsClicked` you can implement it in your XAML code like this:

`<pages:PopupPage
   xmlns:pages="http://rotorgames.com"   xmlns="http://xamarin.com/schemas/2014/forms"
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    CloseWhenBackgroundIsClicked="False">
</pages:PopupPage>`
